### ISSUE TITLE
Update Chrome on Windows vm

### ISSUE THREAD

**Issue by wozzo** [appveyor/ci#2772]
I'd it possible to upgrade the version of Chrome on the Windows vms to the latest?

**Comment by usergenic** [comment]
Did the version downgrade to Chrome 65 recently?  Last week our CI showed it was 69 now its 65 and tests are failing.

**Comment by IlyaFinkelshteyn** [comment]
Please send a link to "before" and "after" builds

**Comment by wozzo** [comment]
@usergenic I've added `choco install googlechrome` to up date it prior to running my tests if that helps

**Comment by usergenic** [comment]
Before: https://ci.appveyor.com/project/Polymer/tools/builds/20833354#L1800
`1800 wct-mocha: chrome 69 (1/0/0)`

After: https://ci.appveyor.com/project/Polymer/tools/builds/21023139#L1773
`1773 wct-browser-legacy: chrome 65                Tests failed:`

FWIW we were using Visual Studio 2013 image.  I've just updated to Visual Studio 2017 image and am watching current build https://ci.appveyor.com/project/Polymer/tools/builds/21026047 to see what it comes up with.

**Comment by usergenic** [comment]
...and it failed.  Chrome 65 on that one too.

**Comment by FeodorFitsner** [comment]
Closing in favor of https://github.com/appveyor/ci/issues/2838
